src/components/WalletModal/index.tsx

```diff
@@ -178,6 +195,13 @@ export default function WalletModal({
       } catch (error) {
         console.debug(`web3-react connection error: ${error}`)
         dispatch(updateConnectionError({ connectionType, error: error.message }))
+
+        // Wallect connection has failed at this point, send failed WALLET_CONNECTED
+        // Amplitude analytics event.
+        sendAnalyticsEvent(EventName.WALLET_CONNECTED, {
```

Comment:
agree, changed to WALLET_CONNECT_TXN_COMPLETED. im against separate event because i think we should make things properties on events where possible (reduce volume of events).